src/main/resources/plugin.yml

```diff
@@ -0,0 +1,23 @@
+name: AdminTools
+version: @version@
+main: com.frostcraft.admintools.Main
+prefix: Frostcraft-AdminTools 
+description: Admin tools for FrostCraft
```

Comment:
If this is targeting 1.13, will it need api-version in plugin.yml?

Reply:
No.

Reply:
Under the notes in the link below, theres a bit on api-version for 1.13. Noticed some messages in another plugin, wasn't sure if would affect this one also. 

https://www.spigotmc.org/threads/bukkit-craftbukkit-spigot-bungeecord-1-13-development-builds.328883/

Reply:
Oh I see, I'll add that and test it out.

Reply:
It still works as intended.